File: fizz_buzz.rb
# ruby fizz_buzz.rb
def get_fizz_buzz(num)
  return nil        unless num.is_a? Numeric
  return "FizzBuzz" if     num % 15 == 0
  return "Fizz"     if     num % 3  == 0
  return "Buzz"     if     num % 5  == 0
  return num
end

ret = (1..100).map {|num| get_fizz_buzz num}
p ret.join ','

File: fizz_buzz.c
// gcc -o fizz_buzz fizz_buzz.c
// ./fizz_buzz
#include <stdio.h>

int main(int argc, char *args[]) {
  for(int i = 1; i <= 100; i++) {
    if (i % 3 == 0) printf("Fizz");
    if (i % 5 == 0) printf("Buzz");
    if (i % 3 != 0 && i % 5 != 0) printf("%d", i);
    printf(",");
  }
  return 0;
}
